dask/dataframe/tests/test_dataframe.py

```diff
@@ -4765,6 +4765,12 @@ def test_nunique(dropna, axis):
     )
 
 
+def test_view():
+    s = pd.Series(range(10), dtype="int8")
+    ds = dd.from_pandas(s, npartitions=2)
+    assert_eq(ds.view("uint8"), s.view("uint8"))
```

Comment:
It think it's also worth adding a test for the original issue in #8439 (that is to say, that we can `view` a datetime without issue)